src/backend/cuda/Array.cpp

```diff
@@ -249,30 +249,25 @@ Array<T> createNodeArray(const dim4 &dims, Node_ptr node) {
                 // some_buffer_size) BUT kernels who's kernel sizes come close
                 // to this value are not passing and cuModuleLoadDataEx is
                 // failing with CUDA_ERROR_INVALID_IMAGE(200). 35*sizeof(int)
-                // seems to be the magic number that passes all tests. I have no
-                // idea why this is the case.
+                // seems to be the magic number that passes all tests.
                 constexpr int max_param_size =
                     (4096 - (sizeof(Param<T>) + 35 * sizeof(uint)));
                 Node *n = node.get();
 
                 struct tree_info {
                     size_t buffer_size;
```

Comment:
This looks like is the size of all buffers under the current jit node. If that is correct, can we rename this appropriately.